x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1192,6 +1192,34 @@ var security = (function () {
         target: "",
     });
 
+    var renameNewProcessFields = new processor.Chain()
+        .Convert({
+            fields: [
+                {from: "winlog.event_data.NewProcessId", to: "process.pid", type: "long"},
+                {from: "winlog.event_data.NewProcessName", to: "process.executable"},
+		{from: "winlog.event_data.ParentProcessName", to: "process.parent.name"},
```

Comment:
Spacing/alignment issue.

Reply:
I think `winlog.event_data.ParentProcessName`  should map to `process.parent.executable`. Then `process.parent.name` should be set to the `basename(process.parent.executable)`.